tests/view_tests/tests/test_default_content_type.py

```diff
@@ -0,0 +1,44 @@
+import sys
```

Comment:
I guess, I thought it was neat naming all the files the same way so they could be quickly found